src/test/java/edu/ucla/library/iiif/fester/handlers/GetManifestHandlerTest.java

```diff
@@ -102,9 +68,9 @@ public void testGetManifestHandlerJson(final TestContext aContext) throws IOExce
     public void testGetManifestHandler404(final TestContext aContext) {
         final Async asyncTask = aContext.async();
         final int port = aContext.get(Config.HTTP_PORT);
-        final String testIDPath = "/testIdentifier"; // path should be: /{id}/manifest
+        final String badPath = "/badIdentifier/manifest"; // path should be: /{id}/manifest
```

Comment:
Perhaps `/missingIdentifier/manifest` or `/404Identifier/manifest`? The ID there isn't bad, it's just not resolvable.